src/flask_pyoidc/flask_pyoidc.py

```diff
@@ -269,8 +249,9 @@ def _logout(self):
         if client.provider_end_session_endpoint:
             flask.session['end_session_state'] = rndstr()
 
+            post_logout_uri = url_for(view_func.__name__, _external=True)
```

Comment:
This will fail for logout view functions that are mounted under blueprints. Name of view functions under a blueprint are prefixed by the name of the associated blueprint. In large-scaled flask apps, this is very common. A user should have an option available to provide view function name that can be resolved by `url_for`.

Reply:
You're right - I opted to leave this behavior for the draft as that's how it was before.

I've just pushed a change now to make it more general, with tests that show it works regardless of which endpoint the view function is mounted under.
The benefit as I see it with this PR is that it will automatically handle the most common cases (including setting the correct `post_logout_redirect_uris` in the registration request) without any changes from the user side.